user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+
+Share grid view
+---------------
+
+The capability enables users to share the :ref:`grid <user-guide-ui-components-grids>` views that they have configured, 
+this way the user can :ref:`adjust a grid <user-guide-ui-components-grids-adjust>` and share it with other users.
+
+This is particularly useful for team-leads and heads of departments, who can share grids with their subordinates, but 
+may also be available to other users.
+
+|
+
+.. image:: ./img/roles/grid_share.png
+
+|
+
+ 
+.. _admin-capabilities-system-info:
+
+System Information
+------------------
+
+The capability enables users to see view the system information page. This page contains the list of Oro Packages and 
+third-party packages installed  and is usually used by the system administrators and integrators only.
+
+
+.. _admin-capabilities-system-config:
+
+System configuration
+--------------------
+
+The capability enables users to access the :ref:`system configuration page <admin-configuration>`. If it is set to 
+*"System"*, the system administrator can localize the system, change the display and tracking settings, and make other 
+important changes.
+
+
+.. _admin-capabilities-tags:
+
+Tag assign/unassign
+-------------------
+
+The capability enables users to assign/unassign :ref:`tags <user-guide-tags>` - non-hierarchical keywords or phrases 
+assigned to records. They provide additional information about records and are visible to all the system users.
+You can assign/unassign tags from the grid or from a View page of the record.
+
+Tags can be successfully utilized by all the users.
+
+
+.. _admin-capabilities-tags-all:
+
+Unassign all tags from entities
+-------------------------------
+
+The capability enables users to unassign tags added to the records by other users. The capability is only 
+meaningful if *"Tag assign/unassign"* has been enabled (set to *"System"*). If *"Tag assign/unassign"* has been enabled, 
+and *"Unassign all tags from entities"* is set to *"None"*, the users will only be able to unassign tags added by 
+themselves -  this way you can restrict users from deleting tags of other users. The capability is usually available to 
+team leads, heads of departments and managers..
+
+.. _admin-capabilities-unshare-grid:
+
+Unshare grid view
+-----------------
+
+If this capability is set to *"System"*, users with the role can unshare grids previously 
+:ref:`shared <admin-capabilities-share-grid>` by themselves. This capability is usually available to all the users, who 
+work with grids.
+
+
+.. _admin-capabilities-view-sql:
+
+View SQL query of a report/segment
+----------------------------------
+
+If this capability is set to *"System"*, users with the role can see the SQL request sent to the system for a 
+report/segment. 
+
+Usually this capability is only granted to the system administrators, so they can check if the report has been developed 
+correctly. The *"Show SQL Query"* link will appear below the report.
```

Comment:
Edit: Usually, this capability is only granted to system administrators so they can check if a report has been developed correctly. The *"Show SQL Query"* link will appear below the report.